pages/common/git-status.md

```diff
@@ -12,10 +12,22 @@
 
 `git status -s`
 
-- Don't show untracked files in the output:
+- Show the [b]ranch and tracking info:
 
-`git status --untracked-files=no`
+`git status -b`
 
 - Show output in [s]hort format along with [b]ranch info:
 
 `git status -sb`
+
+- Show the output in the long-format. This is the default:
+
+`git status --long`
```

Comment:
If this is the default, there's no need to ever use `--long`. I think this should be removed.
```suggestion
```